index.html

```diff
@@ -4100,10 +4105,18 @@ <h2><span class="chunk">cICP</span>
   of computer graphics and web images, including those used in traditional PNG
   workflows, are <a>full-range images</a>.  If <code>Video Full Range
   Flag</code> value is <code>0</code>, then the image is a <a>narrow-range
-  image</a>. Such images are found in video workflows and the interpretation of
-  sample values smaller (larger) than reference black (white) depend on the
-  application. [[SMPTE RP 2077]] specifies mapping between <a>full-range
-  images</a> and <a>narrow-range images</a>.</aside>
+  image</a>. Narrow range images are found in video workflows where the interpretation
+  of sample values below reference black (0% signal level) or above reference white (100% signal level)
+  (E.g. in [[ITU-R BT.709]] 10 bit video, 0% is defined as code value 64 and 100% as code value 940)
+  Such overshoots and undershoots exist to allow momentary excursions in uncontrolled 
+  lighting conditions, preserve processing artifacts caused by video filtering/compression 
+  without clipping which can improve image quality during additional stages of processing 
+  and compression. 
+  
+  The use of undershoot/overshoot has also been used to preserve additional color volume 
+  (both light and color) as described in [[EBU R 103]]. [[SMPTE RP 2077]] describes full range 
```

Comment:
```suggestion
  (both light and color) as described in [[EBU R 103]]. \n [[SMPTE RP 2077]] describes full range 
```

Reply:
I think this isn't the suggested change you had intended.

Reply:
I suggest adding a line break.